### ISSUE TITLE
Download button

### ISSUE THREAD

**Issue by Yuri6037** [nextcloud/spreed#4614]
**Is your feature request related to a problem? Please describe.**
It's annoying to have to open the file view to download a file shared from Nextcloud Talk. Sometimes the view won't even allow to download the file.

**Describe the solution you'd like**
I'd like a download button added to a file block in Nextcloud Talk app. The download button should be a shortcut to the download operation which exists in Nextcloud Files.

**Comment by szaimen** [comment]
https://github.com/nextcloud/viewer/pull/702

**Comment by PVince81** [comment]
in theory a download button could be added on the same level like the reply and the three dots menu for that message ? 

or if that's visually too far away, would need another set of buttons closer to the preview, but that might feel redundant

@ma12-co @jakobroehrl thoughts ?

**Comment by PVince81** [comment]
in the button's implementation, also need to check the file permissions to make sure that download is allowed

**Comment by PVince81** [comment]
slightly related, some other related buttons "copy private link" and/or "jump to files app" https://github.com/nextcloud/spreed/issues/4533

**Comment by ma12-co** [comment]
I agree that it would be nice to have a separate download button for files. Could be something like this? 
![Screenshot from 2021-03-09 11-26-13](https://user-images.githubusercontent.com/26852655/110463862-556eb580-80ca-11eb-80d9-a6400e7e4b75.png)

**Comment by jakobroehrl** [comment]
sure, why not. 
I'd personally perfer the "jump to files app" function for keeping the files in the could

**Comment by ma12-co** [comment]
> I'd personally perfer the "jump to files app" function for keeping the files in the cloud

We could have both in an actions menu

**Comment by nickvergessen** [comment]
Instead of putting it on the image, we could add it into the ... menu with the delete action, jump to files app, etc. because the overlaying only would work with files that have a preview etc. and I prefer to have a single way that works for everything.

**Comment by ma12-co** [comment]
Makes sense too, less hassle with background colors of the button.

**Comment by ChristophWurst** [comment]
![Bildschirmfoto vom 2023-01-17 17-01-47](https://user-images.githubusercontent.com/1374172/212948607-caee21e8-ac09-4951-b637-da64b60b7965.png)

A download button would be most useful for files that can't be viewed

**Comment by fitus85** [comment]
@nickvergessen is it possible to have this issue a bit more on priority? It's really missing in daily use. i think more users are waiting on this.
